### ISSUE TITLE
Drasil Research Group Meeting Agenda, Tuesday Apr 18 at 2:30 pm (on Teams)

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#3331]
The agenda for this coming week is as follows:

1. Jason update
2. Ting-Yu update
3. Sam update
4. Cold cases/Code Review/Discussion
  a. @tingyuw
  b. @cd155
  c. @samm82
  d. @balacij
5. Questions/Comments/Concerns/Discussion

@JacquesCarette likely attending, @tingyuw sends her regrets.

**Comment by JacquesCarette** [comment]
I should be able to make this one.

**Comment by tingyuw** [comment]
I apologize that I won't be able to attend the upcoming meeting. I have a doctor appointment scheduled during that time, and I cannot reschedule it. I'll provide my update here before the meeting next week.

**Comment by smiths** [comment]
We covered all of the agenda items plus a discussion of what information to provide the summer students as part of their "Getting Started" package.

For future meetings (with the summer research students) we discussed presentations/discussions on:

- ChemCode (Sam)
- BeamBending (Jason)
- Review of Paige et al's paper on the problems with MDE

To give everyone time to prepare for the next meeting, we are going to move it from Tuesday to Thursday next week.  I will update the date and time in #3332.
